velox/common/memory/MemoryPool.h

```diff
@@ -1074,3 +1075,14 @@ class StlAllocator {
   }
 };
 } // namespace facebook::velox::memory
+
+template <>
```

Comment:
We can move this inside the `facebook::velox::memory` namespace above and simply use `fmt::formatter<MemoryPool::Kind>`